### ISSUE TITLE
Is BiT truly dead?

### ISSUE THREAD

**Issue by doogie544** [bit-team/backintime#1114]
This was posted in this github in May and there was nothing from any devs that the project was still alive: 
https://github.com/bit-team/backintime/issues/1092  and https://github.com/bit-team/backintime/issues/1088

There is quite a large team by appearances, so I have to wonder: (edit it looks like there were a lot of *contributors* but many of them only had a few commits and it's mostly run by one man) 

Is the project really abandoned?  I know it's one that doesn't need extremely frequent updates but more than a year since the last one seems a bit long even at that.

Does anyone know for sure that the project is dead?  It looks like it but I'm hoping not.

**Comment by Enjymon** [comment]
According to this page there are 3 other persons directly involved:
https://github.com/orgs/bit-team/people

All three are still rather active on their respective projects. Maybe it is worth trying to ask if one of them would be willing to take over the project? I just don't know if this is ok to do, neither how to contact them directly. Does anyone know if there is a private messaging function in GitHub? I could not find any...

**Comment by soulstenance** [comment]
Seems like pretty solid software with a lot of features but it would be great if bugs crop up that someone is there to squash them. Seems the amount of good backup solutions on Linux are few and far between.

**Comment by thwaller** [comment]
I agree with the comments by @soulstenance - there are few backup programs for Linux that are worth talking about. This happens to be one, which I use together with Timeshift. While BiT is functional and works great, there are many improvements that could be made, though not necessary. I would like to see this replace Déjà Dup as a default software for backups, not sure how that process works though.

**Comment by soulstenance** [comment]
Dejadup is an amazing software with an amazing developer who cares about their users. However it's not really targeted at those who like a lot of options. For this reason I use both BiT (for more options) and Dejadup (for stability). I'm not entirely sure how BiT checks for changes but it seems it only checks for file change timestamps which if so isn't the safest way to do this.

**Comment by thwaller** [comment]
@soulstenance I have always found DD hard to use. As this is not the proper place, would you be open to a private dialog on the topic? I al always interested to see how others use software.

**Comment by soulstenance** [comment]
> @soulstenance I have always found DD hard to use. As this is not the proper place, would you be open to a private dialog on the topic? I al always interested to see how others use software.
Sure. Not sure how to do that on here though. You can hit me up on Discord at soulstenance#5424 if you want.

**Comment by colintedford** [comment]
> I'm not entirely sure how BiT checks for changes but it seems it 
only checks for file change timestamps which if so isn't the safest way 
to do this.

@soulstenance BiT does that to save time (possibly it also checks file 
size , or can), but you can set it to check file hashes, and it’s 
recommended to do so occasionally.

**Comment by soulstenance** [comment]
> BiT does that to save time (possibly it also checks file size , or can), but you can set it to check file hashes, and it’s recommended to do so occasionally.

I tried that thinking it was a good idea but it's painfully slow on even a moderate sized backup even if there were no changes. Timeshift for example uses rsync, which I don't fully understand but from what I gather, it's the closest thing Linux has to "volume shadow copy" in Windows and seems like the best option.

**Comment by Codeberg-AsGithubAlternative-buhtz** [comment]
This question/issue comes up from time to time. There are other Issues asking the same question.
The project is not dead.

@Germar could be described as the main developer of backintime. He also take responsibility for quality; means validating PRs and take care of unit-tests etc

He does not work continuously but in waves. ;)

There are other persons around, doing testing or answering questions. Some of the Issues not related to FeatureRequests or Bug but errors on the Users system not related to backintime itself.

**Comment by doogie544** [comment]
Thanks for the answer.  I knew that Germar was the main dev.  I further know that this type of software only needs maintained to keep up with current libs to prevent bugs and does not need frequent releases as it really doesn't need any new features only to be maintained at status quo.  

I only posted my question/issue as all the other ones had never been answered by any of the current team.  It's great to have definitive proof that the project is still maintained and that the users don't need to worry about an unfixed bug making our carefully created backups corrupt or otherwise useless.

Thanks for the work you guys do on this great peace of software.  I have used it several times when distro hopping to keep my important files intact and avoid large downloads of my large steam library after such.  I have also used it to recover from a failed online distro upgrade(glad I left Ubuntu and don't have that as a concern anymore) and have used it's backups many time to browse in and recover a needed file that was corrupted or deleted.
